apps/smart-contracts/core/test/unit/DepositRecord.test.ts

```diff
@@ -363,12 +369,15 @@ describe('=> DepositRecord', () => {
         userDepositBeforeSecondWithdrawal
       )
     })
+
+    after(() => {
```

Comment:
```suggestion
    afterEach(() => {
```
Please do this for all other fake resets as well!

Reply:
It fails with afterEach